### PR TITLE
Fix showing the snippets with description

### PR DESCRIPTION
**Pull request by msivasubramaniaan**
### What does this PR do?

The fix is showing the snippets with corresponding description

### What issues does this PR fix or reference?

https://github.com/redhat-developer/vscode-yaml/issues/642

### Is it tested? How?
Yes with test cases

### CODE REVIEW

**Comment by evidolob** [comment]
@msivasubramaniaan Could you add test for fix?

File: test/schema.test.ts [hunk 1]
```diff
@@ -687,6 +689,20 @@ describe('JSON Schema', () => {
       assert.strictEqual(result.items.length, 1);
       assert.strictEqual(result.items[0].label, 'schemastore');
     });
+
+    it('Default snippet with descritpion', async () => {
```
Comment (JPinkney) [inline test/schema.test.ts:693]:
```suggestion
    it('Default snippet with description', async () => {
```

File: test/schema.test.ts [hunk 2]
```diff
@@ -687,6 +689,20 @@ describe('JSON Schema', () => {
...
+      languageSettingsSetup.withSchemaFileMatch({
+        fileMatch: ['test.yaml'],
+        uri: TEST_URI,
+        priority: SchemaPriority.SchemaStore,
+        schema: schemaDefaultSnippetSample,
+      });
+      languageService.configure(languageSettingsSetup.languageSettings);
+      const testTextDocument = setupTextDocument('foo:  ');
+      const result = await languageService.doComplete(testTextDocument, Position.create(0, 5), false);
+      assert.strictEqual(result.items.length, 2);
```
Comment (JPinkney) [inline test/schema.test.ts:703]: Can you do `assert.equal` on the description value here so that we know it's set correctly

**Review by gorkem — CHANGES_REQUESTED** [review]
Can you please update the author of these commits? Right now they are signed as `unknown` and I am not sure if `<msivasub@win.redhat.com>` is a valid mail address.

**Review by evidolob — APPROVED** [review]

**Pull request merged**